World/Updates/Rel21/Rel21_14_037_S21084SpellChain.sql

```diff
@@ -58,6 +58,7 @@ BEGIN
         -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
 
 -- Insert the correct spell chain for Seal of Righteousness
+DELETE FROM `spell_chain` WHERE `spell_id` IN (20287, 20288, 20289, 20290, 20291, 20292, 20293);
```

Comment:
Remove delete for anything other than 21084 as it is an update which is performed.